src/test/.eslintrc.js

```diff
@@ -16,12 +16,23 @@ module.exports = {
 
     // Adding more errors now
     'testing-library/no-manual-cleanup': 'error',
-    'testing-library/no-wait-for-empty-callback': 'error',
     'testing-library/no-wait-for-snapshot': 'error',
     'testing-library/prefer-explicit-assert': 'error',
     'testing-library/prefer-presence-queries': 'error',
     'testing-library/prefer-wait-for': 'error',
 
+    // Disable some rules that are in the "recommended" part.
+    // This is a purely stylistic rule
+    'testing-library/render-result-naming-convention': 'off',
+    // This disallows using `container`, but this is still useful for us sometimes
+    'testing-library/no-container': 'off',
```

Comment:
I think we could try to stop using `container` and use `document.body` instead, but this seemed like more work than I wanted to do at this point.

Reply:
Yeah, doing this here seems too much work. It could be good to file an issue about that so we can keep track of it.

Reply:
I filed https://github.com/firefox-devtools/profiler/issues/3306